actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -556,10 +570,27 @@ def current_page?(options)
       end
 
       private
+        def default_noopener(rel)
```

Comment:
Breaks if `rel` already have a symbol (`rel: :nofollow`).

could be replaced by something like that:

`rel.to_s.split(' ').push('noopener').uniq`